.github/workflows/ci.yml

```diff
@@ -24,9 +24,7 @@ jobs:
                   coverage: pcov
 
             - name: Checkout
-              uses: actions/checkout@v1
-              with:
-                  fetch-depth: 50
```

Comment:
Why do you fetch only 1 commit (default) and before 50 of them?

Reply:
I had added `fetch-depth: 50` to prevent the whole repo from being cloned. We used the same depth on Travis, so I just copied it.

Apparently, actions/checkout@v2 is smart enough to checkout only the "correct" commit. If so, we would not need to fetch the other 49 commits.

Reply:
👍